src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -126,6 +119,8 @@ public final class FuzzTargetRunner {
     fuzzerTearDown = fuzzTarget.tearDown.orElse(null);
     reproducerTemplate = new ReproducerTemplate(fuzzTargetClass.getName(), useFuzzedDataProvider);
 
+    JazzerInternal.onFuzzTargetReady(targetClassName);
```

Comment:
Is this functionality used / useful anymore?

Reply:
It's not used by Jazzer itself, but I know of third-party hooks that do rely on it (e.g. static analysis developed as part of AI-DevAssist).